src/TicketBai/Mews.Fiscalizations.TicketBai/DtoToModelConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using FuncSharp;
+using Mews.Fiscalizations.TicketBai.Model;
+using System;
+using System.Linq;
+
+namespace Mews.Fiscalizations.TicketBai
+{
+    internal static class DtoToModelConverter
+    {
+        public static SendInvoiceResponse Convert(Dto.TicketBaiResponse response, string qrCodeUri, string xmlRequestContent)
```

Comment:
So we're returning the requestContent into the call site so that the usages can log what was sent right? 
In that case it would also make sense to return the actual request?

Also, where are we getting the QR code from, if not the server response? 😄

Reply:
We will be creating the QR code based on some rules defined in the documentation

its a combination of stuff we define/calculate and stuff returned in the response.

Reply:
But this says qrCodeUri - so it must be somewhere hosted on some URL. So we get sometihng from the response and then upload something somewhere in a second request and get the qr code uri?

Reply:
Anyway, we'll go through the model again later, no need to worry about it now :)